shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/visitor/SQLVisitorRule.java

```diff
@@ -113,15 +113,21 @@
     ALTER_SEQUENCE("AlterSequence", SQLStatementType.DDL),
 
     DROP_SEQUENCE("DropSequence", SQLStatementType.DDL),
-    
+
```

Comment:
@wklxd Please keep indents consistent with the previous one.

Reply:
fix